app/controllers/api/red_hat_migration_analytics_controller.rb

```diff
@@ -37,7 +34,19 @@ def find_provider_ids(type)
     end
 
     class << self
-      def load_manifest(path)
+      def load_manifest
+        # TODO: check for a valid user-provided manifest before defaulting to default-manifest.json
+        manifest_path = Cfme::MigrationAnalytics::Engine.root.join("config", "default-manifest.json")
+        manifest = parse_manifest(manifest_path)
+        {
+          :path => manifest_path,
+          :body => manifest,
+          :version => manifest["manifest"]["version"],
```

Comment:
`manifest.dig("manifest", "version")` is a little safer in case there is no manifest key it won't throw an exception